kratos/tests/test_table_logger.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.kratos_utilities as KratosUtilities
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+class TestTableLogger(KratosUnittest.TestCase):
+
+    def setUp(self):
+        self.comm = KratosMultiphysics.DataCommunicator.GetDefault()
```

Comment:
this is deprecated => use `KratosMultiphysics.Testing.GetDefaultDataCommunicator()` instead